cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -257,7 +257,7 @@ def single_qubit_randomized_benchmarking(
         A RandomizedBenchMarkResult object that stores and plots the result.
     """
 
-    qubits = cast(Iterator['cirq.Qid'], (qubit,))
+    qubits = (qubit,)
```

Comment:
nit: can inline this in the call to `parallel_single_qubit_randomized_benchmarking` below.

Reply:
done